UnitTests/PaymentsCoreTests/NetworkingTests/APIClientTests.swift

```diff
@@ -0,0 +1,78 @@
+import XCTest
+@testable import PaymentsCore
+
+class APIClientTests: XCTestCase {
```

Comment:
```suggestion
class APIClient_Tests: XCTestCase {
```

Also suggestion to rename this file to the same. It matches the other modules test files and I think will be easier to read/sift through as tests grow.